x/erc20/types/msg.go

```diff
@@ -4,23 +4,38 @@
 package types
 
 import (
+	protov2 "google.golang.org/protobuf/proto"
+
 	errorsmod "cosmossdk.io/errors"
 	"cosmossdk.io/math"
+	txsigning "cosmossdk.io/x/tx/signing"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
 
+	erc20api "github.com/evmos/evmos/v19/api/evmos/erc20/v1"
+
 	"github.com/ethereum/go-ethereum/common"
 )
 
 var (
-	_ sdk.Msg = &MsgConvertERC20{}
-	_ sdk.Msg = &MsgUpdateParams{}
+	_ sdk.Msg              = &MsgConvertERC20{}
+	_ sdk.Msg              = &MsgUpdateParams{}
+	_ sdk.Msg              = &MsgRegisterERC20{}
+	_ sdk.Msg              = &MsgToggleConversion{}
+	_ sdk.HasValidateBasic = &MsgConvertERC20{}
+	_ sdk.HasValidateBasic = &MsgUpdateParams{}
+	_ sdk.HasValidateBasic = &MsgRegisterERC20{}
```

Comment:
missing ValidateBasic for `MsgToggleConversion`

Reply:
done, added it